src/main/java/org/fxmisc/flowless/VirtualFlow.java

```diff
@@ -418,4 +448,62 @@ private void jumpToAbsolutePosition(double pixels) {
             navigator.setTargetPosition(new EndOffEnd(items.size() - 1, 0.0));
         }
     }
+
+    /**
+     * The gravity of the virtual flow.  When there are not enough cells to fill
+     * the full height (vertical virtual flow) or width (horizontal virtual flow),
+     * the cells are placed either at the front (vertical: top, horizontal: left),
+     * or rear (vertical: bottom, horizontal: right) of the virtual flow, depending
+     * on the value of the gravity property.
+     *
+     * The gravity can also be styled in CSS, using the "-fx-gravity" property,
+     * for example:
+     * <pre>.virtual-flow { -fx-gravity: rear; }</pre>
+     */
+    public ObjectProperty<Gravity> gravityProperty()
+    {
+        return gravity;
+    }
+
+    public Gravity getGravity()
+    {
+        return gravity.get();
+    }
+
+    public void setGravity(Gravity gravity)
+    {
+        this.gravity.set(gravity);
+    }
+
+    @SuppressWarnings("unchecked") // Because of the cast we have to perform, below
+    private static final CssMetaData<VirtualFlow, Gravity> GRAVITY = new CssMetaData<VirtualFlow, Gravity>(
+            "-fx-gravity",
```

Comment:
I think it's a plausible use case to use Flowless outside RichTextFX.  We could use a Flowless specific prefix, e.g. -flfx-gravity?